src/renderers/dom/shared/eventPlugins/EnterLeaveEventPlugin.js

```diff
@@ -43,8 +43,9 @@ var EnterLeaveEventPlugin = {
     nativeEventTarget,
   ) {
     if (
-      topLevelType === 'topMouseOver' &&
-      (nativeEvent.relatedTarget || nativeEvent.fromElement)
+      (topLevelType === 'topMouseOver' &&
+        (nativeEvent.relatedTarget && !nativeEvent.relatedTarget.disabled)) ||
```

Comment:
`!nativeEvent.relatedTarget.disabled` - is this needed? I think it wont even be emitted by the browser when `relatedTarget` is disabled